ledger/complete/wal/checkpointer_test.go

```diff
@@ -179,6 +180,8 @@ func Test_Checkpointing(t *testing.T) {
 
 				savedData[string(rootHash)] = data
 			}
+			// some buffer time of the checkpointer to run
+			time.Sleep(1 * time.Second)
```

Comment:
Is this to address flakey tests? I'm not too familiar with how this works, but maybe we can expose a way to force the checkpointer to run and wait for it synchronously.